packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -69,6 +69,12 @@ html.interface-interface-skeleton__html-container {
 			padding-bottom: $button-size-small + $border-width;
 		}
 	}
+
+	// Prevent unneccessary infinite empty scrolling when empty block canvas
```

Comment:
What's the infinite empty scrolling? In manual testing on `trunk` it looks like the editor is scrolling nearly a full height of the editor, rather than infinite. Apologies if I'm missing something here!